boards/arm/stm32g0316_disco/doc/index.rst

```diff
@@ -1,6 +1,6 @@
 .. _stm32g0316_disco_board:
 
-STM32G0316-DISCO
+ST STM32G0316 Discovery
 ################
```

Comment:
Need to fix the hash's here

Reply:
The change was in the editor buffer. Fixed, txs!